### PR TITLE
Consent form + UI

### PR DESCRIPTION
**Pull request by OKThomas1**
Added a consent form to the register page as well as some UI fixes.

consent form

The consent checkbox can only be clicked if the user has scrolled to the bottom of the consent form, and the register button can only be clicked if the consent box has been checked.

Additionally, 15px of margin-bottom has been added to the navbar, the django alerts now take up the full width of the container and the text is centered, and an admin nav-item has been added for staff users.

### CODE REVIEW

File: skills_DS/base/templates/base/register.html [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-      </div>
-    </form>
-    <small class="text-muted">Already have an account? <a href="{% url 'login'%}">sign in</a></small>
+  <form method="POST">
+    {% csrf_token %}
+    <fieldset class="form-group">
+      <h2>Register</h2>
+      {{ form|crispy }}
+    </fieldset>
+    <textarea style="height: 400px; width: 400px; overflow-y: scroll" onscroll="onScroll()" id="consentForm" class="form-control">
```
Comment (brianzhouzc) [inline skills_DS/base/templates/base/register.html]: Should probably add `readonly` attribute to prevent users from editing the texts

**Review by brianzhouzc — CHANGES_REQUESTED** [review]
I would add a `readonly` attribute to the textarea to make it read-only. Other than that, everything looks good!

File: skills_DS/base/templates/base/register.html (hunk 1 above)
Comment (OKThomas1) [reply skills_DS/base/templates/base/register.html]: ah, good catch

**Review by brianzhouzc — APPROVED** [review]

**Review by SanaBob — APPROVED** [review]
I would use vh/vw instead of px in navbar. Everything else seems good

**Comment by brianzhouzc** [comment]
I'll merge the pull request then. Furthur styling can be done later when we are styling the entire site.

**Pull request merged**